test/functional/apps/dashboard/group3/bwc_shared_urls.ts

```diff
@@ -75,11 +76,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         log.debug(`Navigating to ${url}`);
         await browser.get(url, true);
         await PageObjects.header.waitUntilLoadingHasFinished();
+        await elasticChart.setNewChartUiDebugFlag(true);
 
         const query = await queryBar.getQueryString();
         expect(query).to.equal('memory:>220000');
 
-        await pieChart.expectPieSliceCount(0);
+        await pieChart.expectEmptyPieChart();
```

Comment:
ℹ️ We don't have EC debug state for empty charts so I am testing this scenario with the empty chart container.